package.json

```
@@ -71,6 +71,6 @@
     "eslint": "^8.6.0",
     "eslint-plugin-header": "^3.1.1",
     "jest": "^27.4.7",
-    "typescript": "^4.5.4"
+    "typescript": "4.5"
```

**Question**: Why do we need to pin TypeScript to 4.5?

---

The updating typescript it yielded in some weird type error for @types/eventsource

---

So we could most likely use the latest patch version? Just to avoid confusion down the road :)

```suggestion
    "typescript": "^4.8.4"
```